systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -675,38 +676,58 @@ void testAutoRenewCaCertsTriggerByExpiredCertificate(ExtensionContext extensionC
 
     @ParallelNamespaceTest
     @Tag(INTERNAL_CLIENTS_USED)
-    void testCertRenewalInMaintenanceWindow(ExtensionContext extensionContext) {
-        final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
-        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
-        final String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
-        final String secretName = KafkaResources.clusterCaCertificateSecretName(clusterName);
-        final String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
+    void testCertRenewalInMaintenanceTimeWindow(ExtensionContext extensionContext) throws InterruptedException {
+        // TODO use testStorage
+        final TestStorage testStorage = new TestStorage(extensionContext, namespace);
+        final String namespaceName = testStorage.getNamespaceName();
+        final String clusterName = testStorage.getClusterName();
+        final String topicName = testStorage.getTopicName();
+        final String userName = testStorage.getUserName();
```

Comment:
I mentioned it in different PR - I don't think that this is the way how to use the `TestStorage`.
The storage contains (almost) everything needed for tests and because we wanted to get rid of all the variables inside the tests, the `TestStorage` was created. Using it this way doesn't make sense, as you are creating new variables, which are same as the ones inside `TestStorage`.

You should go with `TestStorage` and use the getters or use only the variables.

Reply:
Thanks, I wanted to convert test to use TS, but I forgot about it. Good point! Halfway there :-)